imitation/scripts/data_collect.py

```diff
@@ -1,9 +1,6 @@
-import argparse
```

Comment:
Add an observer to the experiment by default. That way it'll save the config, Git commit hashes and other nice things. Without it we're not getting much benefit from switching to Sacred.

Reply:
done, I think